### PR TITLE
Remove nbextensions

### PR DESCRIPTION
**Pull request by damianavila**
Alternative to #5567, let see how it goes...

### CODE REVIEW

File: scripts/travis_install [hunk 1]
```diff
@@ -58,7 +58,9 @@ conda install --yes $DEPS_RUN
 conda install --yes $DEPS_TEST
 
 # conda nb extension is installed by default and breaks notebook image diff tests
-python -m nb_conda_kernels.install --disable --prefix=/home/travis/miniconda
+# python -m nb_conda_kernels.install --disable --prefix=/home/travis/miniconda
+# remove all the nbextension/serverextensions
+conda uninstall --yes _nb_ext_conf
```
Comment (bryevdv) [inline scripts/travis_install]: FWIW when I conda removed the nb extension locally it made my notebook unusable (would not start)

Comment (damianavila) [inline scripts/travis_install]: mmm... that should not happen... any traceback to know why was not working?

**Comment by damianavila** [comment]
It passed... btw, see my comment here: https://github.com/bokeh/bokeh/issues/5541#issuecomment-267449718

**Comment by bryevdv** [comment]
@damianavila you will have to add back the py 2.7 examples tests to `.travis.yml` they were removed to gets tests green.

**Comment by bryevdv** [comment]
It passed because the failing test was not run :)

**Comment by bryevdv** [comment]
You an see how they were added back to the matrix in my other PR

**Comment by damianavila** [comment]
> You an see how they were added back to the matrix in my other PR

OK, added now... let's see...

**Comment by damianavila** [comment]
mmm... nb_conda_kernels was installed from conda-forge after removal... mmm...

**Comment by bryevdv** [comment]
Nope, same problem. Build just "finished" but is not reporting done/green

**Comment by bryevdv** [comment]
This seems like a problem with the extension or with jupyter or both

**Comment by damianavila** [comment]
>This seems like a problem with the extension or with jupyter or both

It could not be the extension... since it is not there... it seems to be some issue with jupyter itself... but only in python 2... interesting.

**Comment by bryevdv** [comment]
Well it was installed at one point so can't rule out that it could be something about it in particular having had been installed

**Pull request closed without merging**